src/bm_sim/actions.cpp

```diff
@@ -20,13 +20,29 @@
  */
 
 #include <bm/bm_sim/actions.h>
+#include <bm/bm_sim/data.h>
```

Comment:
Will assume all of these includes were suggested by our internal tool? Seems fine to me, but would defer to @antoninbas on whether BMv2 uses some other include standards.

Reply:
Thanks for pointing this out.
If something is used in the file, yes I would expect the `#include` to be added to the `.cpp` file even if it is already included in the corresponding header file. So this looks fine to me.
I thought the linter we use in CI was supposed to handle it, but apparently not.